var/spack/repos/builtin/packages/py-luigi/package.py

```diff
@@ -1,9 +1,9 @@
-# Copyright 2013-2021 Lawrence Livermore National Security, LLC and other
+# Copyright 2013-2022 Lawrence Livermore National Security, LLC and other
```

Comment:
```suggestion
# Copyright 2013-2021 Lawrence Livermore National Security, LLC and other
```

Reply:
Had no idea this makes a difference :smile: Thanks!